src/main/pdf/extract.ts

```diff
@@ -46,8 +48,15 @@ export const extractPDFData =
                 // show: false,
                 webPreferences: {
```

Comment:
Why added this in this PR ? I think it's better to create a dedicated commit

Reply:
Thank you Pierre https://github.com/edrlab/thorium-reader/pull/1822/files